olive/model/handler/mixin/hf_config.py

```diff
@@ -91,29 +90,13 @@ def model_loader(_, model_component=model_component):
                 model_attributes=self.model_attributes,
             )
 
-    def load_hf_model(self, model_path: str = None):
```

Comment:
I think moving `get_loading_args` to `hf_utils` make sense but it is better to keep `load_hf_model` as a method here. 
It is only used by the pytorch model handler and is consistent with the other methods here. 

The conversion pass also doesn't use it anymore so a utils is unnecessary.

Reply:
Moved.